Emeryherve/Free-Mentors
# Ft user signin api 168190599

#### WHat this PR does
- Add user sign in API endpoints
#### Description of the task to be completed
- A user is able to sign in using POST /api/v1/auth/signin
#### How should this be tested manually?
1. clone this repo
2. `npm install` all dependencies
3. `npm run test` to see if the endpoints are running
#### PT story
[PT Story](https://www.pivotaltracker.com/story/show/168190599)
#### Any background context
N/A
#### Screenshots
N/A
